### PR TITLE
Add `doc(include = ...)` detection to `missing_docs_in_private_items`

### PR DESCRIPTION
**Pull request by daxpedda**
The whole `missing documentation in crate` part doesn't have any tests. If I should add test cases tell me.

### CODE REVIEW

File: clippy_lints/src/missing_doc.rs [hunk 1]
```diff
@@ -52,6 +53,22 @@ impl MissingDoc {
...
     }
 
+    #[allow(clippy::needless_bool)]
+    fn has_include(meta: Option<MetaItem>) -> bool {
+        if_chain! {
+            if let Some(meta) = meta;
+            if let MetaItemKind::List(list) = meta.node;
+            if let Some(meta) = list.get(0);
+            if let Some(name) = meta.name();
+            if name == "include";
```
Comment (phansch) [inline clippy_lints/src/missing_doc.rs]: I believe this can be rewritten as:

```rust
then {
    name == "include"
}
```

Comment (daxpedda) [reply clippy_lints/src/missing_doc.rs]: Done.
https://github.com/rust-lang/rust-clippy/blob/e1096e3310e414e432884b7b38de8bef26779b7b/clippy_lints/src/missing_doc.rs#L63

**Comment by daxpedda** [comment]
Done.

**Pull request merged**